src/Makefile

```diff
@@ -55,13 +56,13 @@ ifneq ($(BUILD_BENCHMARKS),n)
 	RELEASE_TARGETS += benchmarks
 endif
 
-CLEAN_NO_JE_TARGETS = $(ALL_TARGETS) rpmem_common test
-CLEAN_TARGETS = $(CLEAN_NO_JE_TARGETS) jemalloc examples benchmarks
-CLOBBER_NO_JE_TARGETS = $(ALL_TARGETS) rpmem_common test
-CLOBBER_TARGETS = $(CLOBBER_NO_JE_TARGETS) jemalloc examples benchmarks
-CSTYLE_TARGETS = $(ALL_TARGETS) test rpmem_common examples benchmarks
+CLEAN_NO_JE_TARGETS = $(POSSIBLE_TARGETS)
```

Comment:
this is definitely wrong - CLEAN_NO_JE_TARGETS previously contained "test" and after your patch it doesn't